source/Armament.cpp

```diff
@@ -151,6 +165,11 @@ void Armament::UninstallAll()
 {
 	for(auto &hardpoint : hardpoints)
 		hardpoint.Uninstall();
+
+	turrettedHardpoints.clear();
+	allHardpointsNoAM.clear();
+	fixedHardpoints.clear();
+	antiMissileHardpoints.clear();
```

Comment:
I'm probably blind but where do we fill these?

Reply:
void Armament::RecreateViewsAndRanges()